configuration/index.md

```diff
@@ -292,12 +291,6 @@ gnmi -tls -addr 172.28.161.138:6030 -keyfile gnmi_client.key \
 
 ### NETCONF
 
-[NETCONF](https://tools.ietf.org/html/rfc6241) provides mechanisms to install,
-manipulate and delete the configuration of network devices. It uses eXtensible
-Markup Language (XML) based data encoding for the configuration data as well as
-protocol messages. The NETCONF protocol operations are realized as RPCs over ssh
-transport.
-
```

Comment:
@fredhsu did you mean to remove this text?  it seemed like reasonable expository content.